service/service.go

```diff
@@ -146,3 +146,19 @@ func New(options ...Option) (Owner, error) {
 
 	return svc, nil
 }
+
+type niladicStart func()
```

Comment:
Is niladic a well known word? Slight preference for a simpler word for better readability.